# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/lcms/hotfix-clang.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- lcms-1.19/src/cmsio1.c.vanilla	2023-05-12 23:20:24.533079608 +0200
+++ lcms-1.19/src/cmsio1.c	2023-05-12 23:23:13.847091833 +0200
@@ -22,6 +22,9 @@
 
 // ICC profile serialization
 
+#define _XOPEN_SOURCE       /* See feature_test_macros(7) */
+#include <unistd.h>
+#include <strings.h>
 
 #include "lcms.h"
 
